### ISSUE TITLE
A few interesting lineages on XBB.1.5's messy M:R146H branch

### ISSUE THREAD

**Issue by NkRMnZr** [sars-cov-2-variants/lineage-proposals#562]
```[tasklist]
### A few interesting lineages on XBB.1.5's messy M:R146H branch
- [ ] Branch 1: `XBB.1.5.6 [T17124C > T23018C > G24416A(S:V952I)]` > `G26959A(M:R146H)` > `T10204C` > `C683T` > `G23401T(S:Q613H)` [Query: `T10204C, G23401T, G24416A`, 25 seqs, 08/05]
- [ ] Branch 2: `XBB.1.5 [T17124C > T23018C]` > `G26959A(M:R146H)` > `C13720T(ORF1b:P85S)` > `C3411T(ORF1a:A1049V), C22995G(S:T478R)` [Query: `C3411T, C13720T, C22995G`, 28 seqs, 08/10]
```

### Branch 1

**Defining Mutations:** `XBB.1.5.6 [T17124C > T23018C>G24416A(S:V952I)]` > `G26959A(M:R146H)` > `T10204C` > `C683T` > `G23401T(S:Q613H)`, **EDITED by FEDE**
**Query:** `T10204C, G23401T, G24416A`
**Earliest seq:** 2023-03-13 (EPI_ISL_17345915, Beauvais, France)
**Latest seq:** 2023-07-17 (EPI_ISL_18070072, Gironde, Nouvelle-Aquitaine, France)
**Sampled Countries:** France (21), Italy (3), UK (1, England)

<details><summary>Genomes: </summary>

`EPI_ISL_17345915, EPI_ISL_17480644, EPI_ISL_17480652, EPI_ISL_17491757, EPI_ISL_17502015, EPI_ISL_17547193, EPI_ISL_17548910, EPI_ISL_17638987, EPI_ISL_17639254, EPI_ISL_17671216, EPI_ISL_17671239, EPI_ISL_17677262, EPI_ISL_17693243, EPI_ISL_17705190, EPI_ISL_17705271, EPI_ISL_17769626, EPI_ISL_17781587, EPI_ISL_17792521, EPI_ISL_17813520, EPI_ISL_17833553, EPI_ISL_17859398, EPI_ISL_17949420, EPI_ISL_17958266, EPI_ISL_17976101, EPI_ISL_18070072`

</details>

**UShER:**
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_dcfc_f58b00.json?c=gt-S_613&label=id:node_7063471
![Branch1](https://github.com/sars-cov-2-variants/lineage-proposals/assets/125747944/f0b6dc0b-999f-4ac7-bfd6-333489efcec0)
![Schermata 2023-08-18 alle 13 43 53](https://github.com/sars-cov-2-variants/lineage-proposals/assets/87669813/b8a061cb-cd56-4b88-aa44-55d0e432f6ba)

---

### Branch 2

**Defining Mutations:** `XBB.1.5 [T17124C > T23018C]`  > `C13720T(ORF1b:P85S/nsp12_P94S)` >  `G26959A(M:R146H) > C3411T(ORF1a:A1049V/nsp3_A231V), C22995G(S:T478R)` (**EDITED by FEDE)**
**Query:** `C3411T, C13720T, C22995G`
**Earliest seq:** 2023-02-13 (`EPI_ISL_17027183`, Mississippi, USA)
**Latest seq:** 2023-07-12 (`EPI_ISL_18046455`, Arizona, USA)
**Sampled Countries:** USA (26; Mississippi/23, California/2, Arizona/1)

<details><summary>Genomes: </summary>

`EPI_ISL_17027183, EPI_ISL_17162624, EPI_ISL_17162642-17162643, EPI_ISL_17490158, EPI_ISL_17517881, EPI_ISL_17517885, EPI_ISL_17517904, EPI_ISL_17517941, EPI_ISL_17517946, EPI_ISL_17517950, EPI_ISL_17517956, EPI_ISL_17517972, EPI_ISL_17517992, EPI_ISL_17590099-17590100, EPI_ISL_17590102, EPI_ISL_17624095, EPI_ISL_17624116, EPI_ISL_17682110, EPI_ISL_17682133, EPI_ISL_17707098, EPI_ISL_17744083, EPI_ISL_17744095, EPI_ISL_17768294, EPI_ISL_18046455`

</details>

**UShER:** **NEW** 18-08
https://nextstrain.org/fetch/genome.ucsc.edu/trash/ct/subtreeAuspice1_genome_36b95_f58680.json?c=userOrOld&label=id:node_7248655 **(NEW!)**
![Branch2](https://github.com/sars-cov-2-variants/lineage-proposals/assets/125747944/d7367266-b234-4d3a-8462-0b585e1b00cf)
![Schermata 2023-08-18 alle 13 42 29](https://github.com/sars-cov-2-variants/lineage-proposals/assets/87669813/aac9f838-4b48-46e5-aadb-ed833ee38775)

**Trivia**
- `M:R146H (G26959A)` is one of the definitions of `XAY`
- Many of the sequences have `M:R146H` reverted, if not artifact, could it be some kind of recominant with `XAY`, or different lineages with convergent `M:R146H` happens to be pulled under the branch by UShER? Especially the `T10204C` sub-branch leads to `S:Q613H, S:V952I` listed above, half the sequence having the problem, it's more likely a misplacement to me. Taking look at the `S:V213E>G` > _`M:H146R_reversion`_ > `M:D3N` branch, that looks like a XBB.1.5* with some BA.4/5* recominant, if not mis-placement by UShER.
![R146H](https://github.com/sars-cov-2-variants/lineage-proposals/assets/125747944/93187982-3062-4bb9-93cf-33e2d1858e64)

**Comment by FedeGueli** [comment]
@angiehinrichs please take a look at this one. Could be contamination with XAY? but so much? or real recomb??

cc @ryhisner

**Comment by oobb45729** [comment]
M:R146H does happen at times prior to XAY. Since most sequences of XBB+M:R146H so far are in Asia and Americas. I doubt it has something to do with XAY.

**Comment by AngieHinrichs** [comment]
The late appearance of T10204C in Branch 1 is a big red flag so I pruned the whole XBB.1.5 > 26959 branch and let usher add the sequences back in the 2023-08-17 tree.  Looks like branch 1 sequences were placed in XBB.1.5.6 (S:V952I, nt:G24416A) and branch 2 sequences were moved to XBB.1.5 > ORF1b:P85S, nt:C13720T.  That's in the 2023-08-17 tree, now on dev.usher.bio, on main usher.bio in 4 hours.
